cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -17,23 +17,40 @@
 package co.cask.cdap.api.schedule;
 
 
+import co.cask.cdap.internal.schedule.TimeSchedule;
+
 /**
- * Defines a cron-based schedule for running a program. 
+ * Defines a cron-based schedule for running a program.
  */
 public class Schedule {
 
+  /**
+   * Defines different types of data schedules.
+   */
+  public enum Source {
```

Comment:
Because it is the source of a schedule, so it made more sense to me. But I'll move it.